modules/ossm-release-notes-3-0-new-features.adoc

```diff
@@ -0,0 +1,115 @@
+////
+Module included in the following assemblies:
+* service-mesh-docs-main/ossm-release-notes-assembly.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-0-new-features-enhancements_{context}"]
+= {SMProduct} 3.0 new features and enhancements
+
+This release makes {SMProductName} 3 generally available, adds new features, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {ocp-product-title} 4.14 and later.
```

Comment:
Sometimes, throughout this document, we say "3" and other times we say "3.0". Suggest consistency.